pkg/controller/awsfederatedaccountaccess/awsfederatedaccountaccess_controller.go

```diff
@@ -157,6 +158,35 @@ func (r *ReconcileAWSFederatedAccountAccess) Reconcile(request reconcile.Request
 		return reconcile.Result{}, nil
 	}
 
+	// Check if the FAA has the uuid label
+	if !hasUUIDLabel(currentFAA) {
+		// Generate a new UUID
+		uuid := rand.String(6)
```

Comment:
If we're not using a full 36-char UUID here could we maybe call this something else?  Maybe just UID?  At first glance I was expecting this to be a UUID and not just 6 chars.

Reply:
:+1: Changing